ee/clickhouse/sql/session_recording_events.py

```diff
@@ -16,10 +16,15 @@
     window_id VARCHAR,
     snapshot_data VARCHAR,
     created_at DateTime64(6, 'UTC')
+    {materialized_columns}
     {extra_fields}
 ) ENGINE = {engine}
 """
 
+SESSION_RECORDING_EVENTS_MATERIALIZED_COLUMNS = """
+    , has_full_snapshot BOOLEAN materialized JSONExtractBool(snapshot_data, 'has_full_snapshot') COMMENT 'column_materializer::has_full_snapshot'
```

Comment:
There's some implicit coupling going on here. Worth calling the comment function instead?